test/unit/Cover/editProductTypes.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require('chai');
+
+describe('editProductTypes', function () {
+  it('should edit existing product', async function () {
+    const { cover, accounts } = this;
+
+    const productTypeId = 0;
+
+    const gracePeriodsInDays = [45];
```

Comment:
Passed parameter becomes a 2-dimensional array.

Reply:
Seems to be something to do with how arrays of uints are serialized.

Adding the array nesting or removing it has the same effect.

```cover
        .connect(accounts.advisoryBoardMembers[0])
        .editProductTypes([[productTypeId]], gracePeriodsInDays, [ipfsHash]),```
        
        
       This call where [[productTypeId]] is nested in an array also works correctly.